src/analysis/normed_space/spectrum.lean

```diff
@@ -347,6 +350,34 @@ begin
     (mem_resolvent_set_iff.mp (H₀.symm ▸ set.mem_univ 0)))),
 end
 
+section gelfand_mazur_isomorphism
+
+variables [normed_division_ring A] [normed_algebra ℂ A]
+
+local notation `σ` := spectrum ℂ
+
+open spectrum set
+
+lemma algebra_map_eq_of_mem {a : A} {z : ℂ} (h : z ∈ σ a) : algebra_map ℂ A z = a :=
+by rwa [mem_iff, is_unit_iff_ne_zero, not_not, sub_eq_zero] at h
+
+/-- **Gelfand-Mazur theorem**: For a complex Banach division algebra, the natural `algebra_map ℂ A`
```

Comment:
Strictly this theorem says the isomorphism is also an isometry, right? Is that part of the statement already obvious elsewhere?

Reply:
[algebra_map_isometry](https://leanprover-community.github.io/mathlib_docs/analysis/normed_space/basic.html#algebra_map_isometry)

Reply:
I added a bit to the docstring referencing the isometry fact. Note: currently we don't have things like continuous algebra homomorphisms or isometric algebra homomorphisms as bundled maps, so it is impossible for us to state the whole theorem at once. (Also, this map is a star isomorphism too, but we can't say that either yet I don't think).